cmd/qat_plugin/README.md

```diff
@@ -245,10 +258,190 @@ You can verify this on your nodes by checking for the relevant PCI identifiers:
 for i in 0442 0443 18a1 37c9 6f55 19e3 4941 4943; do lspci -d 8086:$i; done
 ```
 
+## NOTES
+1. The initcontainer deployment will split the qat resources into two: cy and dc.
+2. cy stands for crypto and dc for compression. Depending on what your workload needs, you can then request the either for the container.
+
+
```

Comment:
2. is documented in "automatic provision" table. 1. is not completely true: the "split" is the default from the driver so the initcontainer has no role in that.